DQMOffline/Trigger/python/TopMonitoring_cff.py

```diff
@@ -1,4 +1,193 @@
 import FWCore.ParameterSet.Config as cms
 
+from DQMOffline.Trigger.TopMonitor_cfi import hltTOPmonitoring
+
+eleJet_jet = hltTOPmonitoring.clone()
+eleJet_jet.FolderName = cms.string('HLT/TopHLTOffline/TopMonitor/Top/EleJet/JetMonitor')
+eleJet_jet.nmuons = cms.uint32(0)
+eleJet_jet.nelectrons = cms.uint32(1)
+eleJet_jet.njets = cms.uint32(1)
+eleJet_jet.eleSelection = cms.string('pt>50 & abs(eta)<2.1 & (dr03TkSumPt+dr04EcalRecHitSumEt+dr04HcalTowerSumEt)/pt<0.1')
+eleJet_jet.jetSelection = cms.string('pt>30 & abs(eta)<2.4')
+eleJet_jet.histoPSet.eleEtaBinning = cms.vdouble(-2.1,-1.7,-1.2,-0.9,-0.6,-0.3,-0.1,0,0.1,0.3,0.6,0.9,1.2,1.7,2.1)
+eleJet_jet.histoPSet.eleEtaBinning2D = cms.vdouble(-2.1,-1.7,-1.2,-0.6,-0.3,0,0.3,0.6,1.2,1.7,2.1)
+eleJet_jet.histoPSet.elePtBinning = cms.vdouble(50,60,80,120,200,400)
+eleJet_jet.histoPSet.elePtBinning2D = cms.vdouble(50,70,120,200,400)
+eleJet_jet.histoPSet.jetPtBinning = cms.vdouble(30,32.5,35,37.5,40,45,50,60,80,120,200,400)
+eleJet_jet.histoPSet.jetPtBinning2D = cms.vdouble(30,35,40,45,50,60,80,100,200,400)
+eleJet_jet.numGenericTriggerEventPSet.hltPaths = cms.vstring('HLT_Ele30_eta2p1_WPTight_Gsf_CentralPFJet35_EleCleaned_v*')
+eleJet_jet.denGenericTriggerEventPSet.hltPaths = cms.vstring('HLT_Ele30_eta2p1_WPTight_Gsf_v*',
+                                                             'HLT_Ele35_WPTight_Gsf_v*',
+                                                             'HLT_Ele38_WPTight_Gsf_v*',
+                                                             'HLT_Ele40_WPTight_Gsf_v*',)
+
+eleJet_ele = hltTOPmonitoring.clone()
+eleJet_ele.FolderName = cms.string('HLT/TopHLTOffline/TopMonitor/Top/EleJet/ElectronMonitor')
+eleJet_ele.nmuons = cms.uint32(0)
+eleJet_ele.nelectrons = cms.uint32(1)
+eleJet_ele.njets = cms.uint32(1)
+eleJet_ele.eleSelection = cms.string('pt>25 & abs(eta)<2.1 & (dr03TkSumPt+dr04EcalRecHitSumEt+dr04HcalTowerSumEt)/pt<0.1')
+eleJet_ele.jetSelection = cms.string('pt>50 & abs(eta)<2.4')
+eleJet_ele.histoPSet.eleEtaBinning = cms.vdouble(-2.1,-1.7,-1.2,-0.9,-0.6,-0.3,-0.1,0,0.1,0.3,0.6,0.9,1.2,1.7,2.1)
+eleJet_ele.histoPSet.eleEtaBinning2D = cms.vdouble(-2.1,-1.7,-1.2,-0.6,-0.3,0,0.3,0.6,1.2,1.7,2.1)
+eleJet_ele.histoPSet.elePtBinning = cms.vdouble(25,27.5,30,32.5,35,40,45,50,60,80,120,200,400)
```

Comment:
@mtosi  I could, but the offline cut is at 25 for electrons and 50 for jets. It would only mean having empty bins